index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <link rel="stylesheet" href="styles.css"/>
+</head>
+<body>
+    <!---PROJECT HEADER SECTION-->
+    <header> 
+        <!--TOOLBAR-->
+            <div class="toolbar-container">
+                <div class="logo-container">
+                     <a href="" class="logo">
+                        <h3>Welcome</h3>
+                     </a>
+                     <button type="button" class="menu-btn">
+                        <img src="./img/Union.svg" alt="menu">
+                     </button>
+                     <nav class="header-toolbar hidden">
+                        <button type="button" class="close-btn">
+                            <img src="./img/Icon.svg" alt="cancel">
+                        </button>
+                        <ul>
+                            <li><a href="#" class="">Portfolio</a></li>
+                            <li><a href="#" class="">About</a></li>
+                            <li><a href="#" class="">Contact</a></li>
+                        </ul>
+                     </nav>
+                </div>
+            </div>
+    </header>
+    <!--HEADLINE-->
+    <section class="headline-section">
+        <div class="container-flex">
+            <h1 class="headline-title">Hey there. I'm Rachel</h1>
+            <h1><span>I'm a software developer</span></h1>
```

Comment:
- [ ] it is more useful to use a single `h1` tag for every single webpage for semantic improvement. kindly change the subtitle to a `p`.